api/scpca_portal/management/commands/load_data.py

```diff
@@ -24,6 +24,8 @@
     "murphy_chen",
     "pugh",
     "teachey_tan",
+    "wu",
+    "rokita",
```

Comment:
```suggestion
    "rokita"
```

Reply:
This might be convention based on the diff, so feel free to ignore me!

Reply:
Yeah it's convention so we don't have to edit the last item's line when adding/sorting a list. Our linter does require it.